storybook/qmlTests/tests/tst_AssetSelector.qml

```diff
@@ -0,0 +1,238 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtTest 1.15
+
+import AppLayouts.Wallet.controls 1.0
+
+import utils 1.0
+
+Item {
+    id: root
+
+    width: 600
+    height: 600
+
+    readonly property var assetsData: [
+        {
+            tokensKey: "stt_key",
+            communityId: "",
+            name: "Status Test Token",
+            currencyBalanceAsString: "42,23 USD",
+            symbol: "STT",
+            iconSource: Constants.tokenIcon("STT"),
+
+            balances: [
+                {
+                    balanceAsString: "0,56",
+                    iconUrl: "network/Network=Ethereum"
+                }
+            ],
+
+            sectionText: "My assets on Mainnet"
+        },
+        {
+            tokensKey: "eth_key",
+            communityId: "",
+            name: "Ether",
+            currencyBalanceAsString: "4 276,86 USD",
+            symbol: "ETH",
+            iconSource: Constants.tokenIcon("ETH"),
+
+            balances: [
+                {
+                    balanceAsString: "0,12",
+                    iconUrl: "network/Network=Ethereum"
+                }
+            ],
+
+            sectionText: "My assets on Mainnet"
+        },
+        {
+            tokensKey: "dai_key",
+            communityId: "",
+            name: "Dai Stablecoin",
+            currencyBalanceAsString: "45,92 USD",
+            symbol: "DAI",
+            iconSource: Constants.tokenIcon("DAI"),
+            balances: [],
+
+            sectionText: "Popular assets"
+        }
+    ]
+
+    ListModel {
```

Comment:
I think it's a good approach to embed the modifiable dependencies in the `Component` under test. This way we're ensuring that each test runs in isolation even if we need to update this model for some specific tests.

Or another option would be to add such dependencies in `Component` and to create it when needed.

WDYT?

Reply:
Yes, I'm also for that rule, for that reason I always put `SignalSpy`s into `Component`. I left the model outside only because it's considered as static test data here.

We could go bit further and always, preventively put  it inside, as you suggest. I can do that if you think it's good to keep it there always.